client/lib/performance-tracking/api.js

```diff
@@ -0,0 +1,43 @@
+const getLux = () => {
+	const lux = window.LUX || {};
+	// Reassign it so changes in LUX are picked when Lux starts
+	window.LUX = lux;
+
+	// Don't send the first navigation automatically, we'll do it.
+	lux.auto = false;
+	lux.debug = true;
+	return lux;
+};
+
+const addMetadata = ( metadata ) => {
+	const lux = getLux();
+
+	Object.entries( metadata ).forEach( ( [ key, value ] ) => {
```

Comment:
Relevant reads:

- https://support.speedcurve.com/en/articles/1262334-lux-customer-data

It is worth mentioning that for this to work, we have to manually create each variable in Speedcurve's UI as well.